lecture_notes/Updating_files_with_github.Rmd

```diff
@@ -0,0 +1,33 @@
+---
+title: "Updating files on GitHub"
+author: "Data Wrangling in R"
+output:
+  ioslides_presentation:
+    css: styles.css
+    widescreen: yes
+---
+
+```{r, echo = FALSE, include=FALSE, purl=FALSE}
+library(knitr)
+opts_chunk$set(comment = "")
+```
+
+```{r, include=FALSE}
+library(tidyverse)
+```
+
+## Updating files
+
+Remember, we want to update files on AnVIL if there are changes in the remote repository!
+
```

Comment:
should we say something about there notes and how they can export them?

Reply:
but otherwise looks good :)

Reply:
Yes good call!